pkgs/by-name/os/ossia-score/package.nix

```diff
@@ -0,0 +1,162 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, ninja
+, pkg-config
+, alsa-lib
+, avahi
+, avahi-compat
+, bluez
+, boost185
+, fmt
+, ffmpeg
+, fftw
+, flac
+, git
+, gnutls
+, lame
+, libcoap
+, libjack2
+, libopus
+, libsamplerate
+, libsndfile
+, libvorbis
+, lilv
+, lv2
+, mpg123
+, pipewire
+, portaudio
+, qt6
+, rapidfuzz-cpp
+, re2
+, rubberband
+, snappy
+, SDL2
+, spdlog
+, suil
+, udev
+}:
+
+# TODO: figure out LLVM jit
+# assert lib.versionAtLeast llvm.version "15";
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "ossia-score";
+  version = "3.2.2";
+
+  src = fetchFromGitHub {
+    owner = "ossia";
+    repo = "score";
+    rev = "v3.2.2";
+    hash = "sha256-WwP21wZf1rVKIRiW3THlwbO/L6pgihWmBJx52EvMWHw=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ cmake ninja pkg-config qt6.qttools qt6.wrapQtAppsHook ];
+
+  buildInputs = [
+    alsa-lib
+    boost185
+    avahi
+    avahi-compat
+    bluez
+    ffmpeg
+    fftw
+    flac
+    fmt
+    git
```

Comment:
But I guess we don't need this `git` or should put it in `nativeBuildInputs`?

It's used like this: https://github.com/ossia/libossia/blob/34f7737330f195fdb67214ea7f9a6cc038deaa0c/cmake/OssiaDeps.cmake#L70
```cmake
  execute_process(COMMAND ${GIT_EXECUTABLE} submodule sync --recursive
                  WORKING_DIRECTORY ${CMAKE_CURRENT_LIST_DIR}
                  COMMAND_ERROR_IS_FATAL ANY)
```

Reply:
Ah true,sorry I don't know exactly what the difference between these two keys is and just put it where it was needed to fix the build